3.3/test/run

```diff
@@ -106,6 +106,43 @@ test_docker_run_usage() {
   docker run ${IMAGE_NAME} &>/dev/null
 }
 
+test_scl_usage() {
+  local run_cmd="$1"
+  local expected="$2"
+  local cid_file="$3"
+
+  info "Testing the image SCL enable"
+  out=$(docker run --rm ${IMAGE_NAME} /bin/bash -c "${run_cmd}" 2>&1)
+  if ! echo "${out}" | grep -q "${expected}"; then
+    echo "ERROR[/bin/bash -c "${run_cmd}"] Expected '${expected}', got '${out}'"
+    return 1
+  fi
+  out=$(docker run --rm -i ${IMAGE_NAME} /bin/bash -i <<< "${run_cmd}" 2>&1)
+  if ! echo "${out}" | grep -q "${expected}"; then
```

Comment:
perl gives you multi line output and exec do some crazy stuff with ioctl so I would rather keep it with grep as that works ;-) we can fix as follow up or when we will be reworking the tests in post-3.0